doc/howto/troubleshoot.md

```diff
@@ -8,6 +8,7 @@ If you run into problems when using LXD, check the following how-to guides to se
 
 Configure your firewall </howto/network_bridge_firewalld>
 Troubleshoot instances </howto/instances_troubleshoot.md>
+Troubleshoot dqlite </howto/dqlite_troubleshoot.md>
```

Comment:
Can you add the same line before this one?
https://github.com/canonical/lxd/blob/3fef7cb76eb98a41075c8398c178bd9f45079f68/doc/operation.md?plain=1#L42
Or actually, not the same but with `topical:` added. ;)